.github/workflows/runtimeCheck.yml

```diff
@@ -20,6 +20,14 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
+            comments = await github.rest.issues.listComments({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number
+            });
+            if (comments.data.some(comment => comment.body.includes('This pull request contains changes to the runtime.ts file.'))) {
```

Comment:
>'This pull request contains changes to the runtime.ts file.' [](http://example.com/codeflow?start=68&length=60)

Can you extract this statement into a single constant so it can be used both here and on line 35? Does YAML allow that?

Reply:
Good question... That's something I haven't seen but could take a look

Reply:
I think it could be defined as a variable(?) at the top of file and then be used later

Reply:
Good suggestion, I'll give it a try

Reply:
Ok, I just used a variable in the script, but they're using the same string now